app/org/maproulette/framework/controller/TaskBundleController.scala

```diff
@@ -85,14 +83,6 @@ class TaskBundleController @Inject() (
       for (task <- tasks) {
         val action = this.actionManager
           .setAction(Some(user), new TaskItem(task.id), TaskStatusSet(status), task.name)
-        // add comment to each task if any provided
-        if (comment.nonEmpty) {
```

Comment:
Isn't this needed since it's adding the provided comment to each task?

Reply:
After discussion, it appears that this would sync the OSM Comment with the MapRoulette tasks (users would get a MR notification). However this use case hasn't been used in at least 2 years. We're in agreement to remove this.